src/codeu/chat/client/ClientConversation.java

```diff
@@ -76,7 +76,7 @@ public ConversationSummary getCurrent() {
     return currentSummary;
   }
 
-  public Uuid getCurrentId() { return (currentSummary != null) ? currentSummary.id : null; }
+  public String getCurrentId() { return (currentSummary != null) ? currentSummary.id : null; }
```

Comment:
It's probably going to be a lot better on the merge conflict front if you simply adapt the Uuid class to your needs rather than changing to string completely.  Maybe a FirebaseUUID class that extends UUID but with the content stored in a string.